modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Environment.scala

```diff
@@ -13,34 +13,26 @@
 package com.snowplowanalytics.snowplow.enrich.common.fs2
 
 import scala.concurrent.duration.FiniteDuration
-
 import cats.Show
 import cats.data.EitherT
 import cats.implicits._
-
 import cats.effect.{Async, Blocker, Clock, ConcurrentEffect, ContextShift, ExitCase, Resource, Sync, Timer}
 import cats.effect.concurrent.{Ref, Semaphore}
-
 import fs2.Stream
-
 import _root_.io.sentry.{Sentry, SentryClient}
-
 import org.http4s.client.{Client => Http4sClient}
 import org.http4s.Status
 import org.typelevel.log4cats.Logger
 import org.typelevel.log4cats.slf4j.Slf4jLogger
-
 import com.snowplowanalytics.iglu.client.IgluCirceClient
 import com.snowplowanalytics.iglu.client.resolver.registries.{Http4sRegistryLookup, RegistryLookup}
-
 import com.snowplowanalytics.snowplow.badrows.Processor
 import com.snowplowanalytics.snowplow.enrich.common.adapters.AdapterRegistry
 import com.snowplowanalytics.snowplow.enrich.common.adapters.registry.RemoteAdapter
 import com.snowplowanalytics.snowplow.enrich.common.enrichments.EnrichmentRegistry
 import com.snowplowanalytics.snowplow.enrich.common.enrichments.registry.EnrichmentConf
 import com.snowplowanalytics.snowplow.enrich.common.outputs.EnrichedEvent
 import com.snowplowanalytics.snowplow.enrich.common.utils.{BlockerF, HttpClient, ShiftExecution}
-
 import com.snowplowanalytics.snowplow.enrich.common.fs2.config.{ConfigFile, ParsedConfigs}
```

Comment:
Do you have a formatter set up ? We ususally like it to separate the packages by lib with an empty line in-between

Reply:
Sorry I didn't realize there was a convention for this, my IDE had a setting to reorder the imports alphabeticallly. Have reverted it now!